inbm/cloudadapter-agent/cloudadapter/cloud/client/cloud_client.py

```diff
@@ -13,18 +13,21 @@
 class CloudClient:
 
     def __init__(self, connection: MQTTConnection, telemetry: OneWayMessenger, event: OneWayMessenger,
-                 attribute: OneWayMessenger, handler: ReceiveRespondHandler) -> None:
+                 update: OneWayMessenger, attribute: OneWayMessenger, 
+                 handler: ReceiveRespondHandler) -> None:
         """Constructor for CloudClient
 
         @param connection: Connection associated with this CloudClient
         @param telemetry: Messenger to send telemetry
         @param event: Messenger to send events
+        @param update: Messenger to send updates
```

Comment:
`update` => `node_update` and `Messenger to send updates` => `Messenger to send node updates`